### ISSUE TITLE
costHydraulicSystemsType missing cost element in costDescription

### ISSUE THREAD

**Issue by DerFriese1992** [DLR-SL/CPACS#811]
Hi everyone,

It looks like the costHydraulicSystemsType differs from all the others:
![grafik](https://user-images.githubusercontent.com/54900300/220556436-9ae314c4-3976-4a72-b48f-fdff94a032ad.png)

While the other cost types have a costDescription which consists of a genericCostType like this:
![grafik](https://user-images.githubusercontent.com/54900300/220557291-8c6a5705-4e84-4bf8-9486-2c7375926e3b.png)

The costHydraulicSystemsType has a costDescription which only consists of a string:
![grafik](https://user-images.githubusercontent.com/54900300/220556881-9f13b666-d014-41d3-a24f-6a3cac6de0e9.png)

In short, you can only put in a single string into the hyraulic systems costs.

**Comment by MarAlder** [comment]
Thanks @DerFriese1992 for the bug report. We'll fix it for the next release.
